packages/legend-application-query/src/stores/QueryBuilderResultState.ts

```diff
@@ -172,9 +181,9 @@ export class QueryBuilderResultState {
     }
   }
 
-  *execute(): GeneratorFn<void> {
+  async runQuery(): Promise<void> {
```

Comment:
Can we leave this as `generator` here. I'm worried since we're relying on `makeAutoObservable` to pick this up as `asyncAction`, but we should properly use `flow` here instead